ansible/includes/redirect-and-acme-challenge-includes.yml

```diff
@@ -17,8 +17,12 @@
       <Directory /var/lib/acme/challenges>
           Require all granted
       </Directory>
+      <location /.well-known/acme-challenge/>
+          Require all granted
+      </location>
```

Comment:
This `<location>` block is not necessary because the access rights are granted by the `<Directory>` above and the `Alias` points to that directory.